### PR TITLE
Update mpHealth-2.2 API binaries to official release spec

### PR DESCRIPTION
**Pull request by pgunapal**

### CODE REVIEW

**Comment by pgunapal** [comment]
Fixes #10823

**Comment by pgunapal** [comment]
#build

**Comment by pgunapal** [comment]
#build

File: dev/cnf/oss_dependencies.maven [hunk 1]
```diff
@@ -316,6 +316,7 @@ org.eclipse.microprofile.graphql:microprofile-graphql-api:1.0-M3
 org.eclipse.microprofile.health:microprofile-health-api:2.0.1
 org.eclipse.microprofile.health:microprofile-health-api:2.1
 org.eclipse.microprofile.health:microprofile-health-api:2.2-RC2
```
Comment (fmhwong) [inline dev/cnf/oss_dependencies.maven]: Can RC2 be removed?

Comment (pgunapal) [reply dev/cnf/oss_dependencies.maven]: I would have to remove it, once i merge this PR, since I have to delete the binaries in Artifactory as well.

**Comment by pgunapal** [comment]
#run-libby-bot

**Pull request merged**